geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/stats/BinnedArray.scala

```diff
@@ -125,10 +127,10 @@ object BinnedArray {
 
 class BinnedIntegerArray(length: Int, bounds: (Integer, Integer)) extends BinnedArray[Integer](length, bounds) {
 
-  require(bounds._1 < bounds._2,
+  require(bounds._1 <= bounds._2,
     s"Upper bound must be greater than lower bound: lower=${bounds._1} upper=${bounds._2}")
```

Comment:
actually i'm going to change it back to strictly less than - for a while i thought i needed equals but not any more.